### PR TITLE
added tests for -Z switch for shp2pgsql

### PR DESCRIPTION
**Pull request by wsdookadr**

### CODE REVIEW

**Comment by robe2** [comment]
It looks good to me but what do I know :)

I'll give @strk one day to comment before I commit.

File: regress/run_test.pl [hunk 1]
```diff
@@ -970,6 +970,14 @@ sub run_loader_and_check_output
 			}
 		}
 
+		if ( $loader_options =~ m{^-Z } ) {
```
Comment (strk) [inline regress/run_test.pl:973]: Does this not only match `-Z` if given as the very first option ?

**Review by strk — COMMENTED** [review]
Looks good to me, but there's no actual test checking for -Z ?

File: regress/run_test.pl (hunk 1 above)
Comment (wsdookadr) [reply regress/run_test.pl:973]: Yes. I propose we replace that with `m{^-Z|\s-Z\s}` to cover all the valid locations:

```bash
echo -e "-Z a b\na -Z b\na b -Z\na b c\na-Z b c\na b c-Z" | perl -ne 'print if m{^-Z|\s-Z\s};'
```

output:
```
-Z a b
a -Z b
a b -Z
```

**Comment by wsdookadr** [comment]
Closing in favor of #612

**Pull request closed without merging**